### DISCUSSION TITLE
Clear visual distinction between missing the goal vs not marking it yet

### DISCUSSION THREAD

**Discussion by MarcinOrlowski** [iSoron/uhabits#805] [Feature Requests] [answered]
For each habit we have two states only: either is checked (green) or not checked (default). Some other trackers I checked offer 3rd state: "missed" (explicit). So item have to be clearly marked as missed instead of being considered missed by-default as uhabits currently handles this. It's useful esp. you want to fill some "backlog" then you can clearly tell which days haven't been recorded yet.

**Comment by hiqua** [comment, marked as answer]
It's implemented in the v2 (see https://github.com/iSoron/uhabits/releases/tag/v2.0.0-alpha).

**Reply by MarcinOrlowski** [reply]
Thanks. Any ETA for official release of 2.0? Also this alpha is not installable over existing version

> Failure [INSTALL_FAILED_UPDATE_INCOMPATIBLE: Package org.isoron.uhabits signatures do not match previously installed version; ignoring!]

Wouldn't it be more useful to release alpha with different packageId?

**Reply by hiqua** [reply]
No there's no ETA since only volunteers work on this. You can look at what's left in the milestone to have an idea: https://github.com/iSoron/uhabits/milestone/11

Alpha versions are not meant to be easily installable, so although I agree with you, it's almost a feature here.

**Reply by MarcinOrlowski** [reply]
I just planted old DB and it worked (I could probably also import the backup ;). 

May I complain that the "x" mark has the same color as the "?". Can that be configurable? I understand the tick is using habit color, so "x" could either use gray as now (default) or color set by user.
